md/_resources.md

```diff
@@ -33,5 +33,5 @@ Everything you need to become a Privacy Wizard.
 - [Guide for Linux users](https://github.com/wesaphzt/block-all-google) on blocking everything Google at the network level. This can and probably will break other alternatives that rely on Google to serve content. Thanks u/wesaphzt
 - u/wesaphzt also provided [this](https://github.com/pyllyukko/user.js), [this](https://ffprofile.com/), and [this](https://www.privacytools.io/browsers/#about_config) for hardening Firefox.
 - Check out [WindowsSpyBlocker](https://github.com/crazy-max/WindowsSpyBlocker) and [this tutorial](https://github.com/adolfintel/Windows10-Privacy) for even more Windows 10 tips (thanks u/rudolf323)
-- [Virustotal](https://www.virustotal.com/) let's you upload files and it scans it against multiple antivirus software at once. Very helpful for checking false negatives. Website uses Google Analytics.
-- Did you read the [wiki](https://old.reddit.com/r/privacy/wiki/index) yet? Go do that.
\ No newline at end of file
+- [Virustotal](https://www.virustotal.com/) let you upload files and it scans it against multiple antivirus software at once. Very helpful for checking false negatives. Unfortunately, Alphabet Inc owns VirusTotal, but there are no good alternatives.
```

Comment:
```suggestion
- [VirusTotal](https://www.virustotal.com/) lets you upload files and scans it against multiple antivirus programs at once. Very helpful for checking false negatives. Unfortunately Alphabet Inc owns VirusTotal but there are no good alternatives.
```
I'm not a native speaker but isn't this better?
Otherwise approved.